doc/sedcli.8

```diff
@@ -58,9 +58,17 @@ Activates Locking SP.
 .IP "\fB\-R -d <device> [-i]\fR or \fB\-\-revert --device <device> [--psid]\fR"
 Reverts TPer to Manufactured-Inactivate state using either SID or PSID authority.
 
-.IP "\fB\-L -d <device> -t <locktype>\fR or \fB\-\-lock-unlock --device <device> --locktype {RW|RO|WO|UNLOCK}\fR"
+.IP "\fB\-L -d <device> -t <locktype>\fR or \fB\-\-lock-unlock --device <device> --locktype {RO|RW|LK}\fR"
```

Comment:
suggest to change the name of the parameter from locktype to accesstype.

Reply:
People already familiar with the opal parameters or driver code may find it confusing with the nomenclature if we change it

Reply:
We are not changing kernel naming. We propose utility and library that doesn't have the same confusion like kernel has.